src/main/java/com/thevoxelbox/voxelsniper/util/message/Messenger.java

```diff
@@ -45,7 +45,7 @@ public void sendReplaceBlockDataMessage(BlockState replaceBlockData) {
 
     public void sendBrushSizeMessage(int brushSize) {
         sendMessage(ChatColor.GREEN + "Brush Size: " + ChatColor.DARK_RED + brushSize);
```

Comment:
Is the input validated somewhere? Currently you can use brushes with a size lesser than 0, where 0 is already the smallest size you can have.

Reply:
Should be validated in ``BrushExecutor.java`` and in config loader, this line just takes a valid input to print it.